src/core_plugins/kibana/public/visualize/wizard/wizard.js

```diff
@@ -21,13 +22,21 @@ routes.when('/visualize/step/1', {
   template: templateStep(1, require('plugins/kibana/visualize/wizard/step_1.html'))
 });
 
-module.controller('VisualizeWizardStep1', function ($scope, $route, $location, timefilter, Private) {
+module.controller('VisualizeWizardStep1', function ($scope, $route, kbnUrl, timefilter, Private) {
   timefilter.enabled = false;
 
+  $scope.addToDashMode = $route.current.params[DashboardConsts.ADD_TO_DASH_PARAM];
```

Comment:
If the template doesn't need access to `addToDashMode` then this should just be a `const` instead of a scope variable.